app/util/bitbucket/populate_db.sh

```diff
@@ -94,7 +94,30 @@ else
   echo "Postgres client is already installed"
 fi
 
-echo "Step2: Download DB dump"
+echo "Step2: Get DB Host"
+DB_HOST=$(sudo su -c "cat ${DB_CONFIG} | grep 'jdbc:postgresql' | cut -d'/' -f3 | cut -d':' -f1")
+if [[ -z ${DB_HOST} ]]; then
+  echo "DataBase URL was not found in ${DB_CONFIG}"
+  exit 1
+fi
+echo "DB_HOST=${DB_HOST}"
+
+echo "Step3: Write 'instance.url' property to file"
+BITBUCKET_BASE_URL_FILE="base_url"
+if [[ -s ${BITBUCKET_BASE_URL_FILE} ]]; then
+  echo "File ${BITBUCKET_BASE_URL_FILE} was found. Base url: $(cat ${BITBUCKET_BASE_URL_FILE})."
+else
+  PGPASSWORD=${BITBUCKET_DB_PASS} psql -h ${DB_HOST} -d ${BITBUCKET_DB_NAME} -U ${BITBUCKET_DB_USER} -c \
+  "select prop_value from app_property where prop_key='instance.url';" |
+  awk '/.htt/{print $1}' > ${BITBUCKET_BASE_URL_FILE}
+  if [[ ! -s ${BITBUCKET_BASE_URL_FILE} ]]; then
+    echo "Failed to get Base URL value form database. Check DB configuration variables."
+    exit 1
+  fi
+  echo "$(cat ${BITBUCKET_BASE_URL_FILE}) was written to the ${JIRA_BASE_URL_FILE} file."
```

Comment:
JIRA_BASE_URL_FILE in populate.db for bitbucket

Reply:
Yes, good catch! Thank you.